.github/workflows/installer-macos.yml

```diff
@@ -9,7 +9,7 @@ name: Create macOS App Bundle and DMG
 jobs:
   build:
     name: macOS App Bundle
-    runs-on: macos-latest
+    runs-on: macos-11.0
```

Comment:
I think we should add a new workflow for Big Sur. Otherwise, we'll affect people still running Catalina.

Reply:
Agreed. This is just for testing right now. When this becomes available, and we can build on Apple silicon, then we can consider adding separate workflows or expand the matrix workflow; in any case we should always have a Catalina build